pkg/app/daemon/daemon.go

```diff
@@ -27,8 +27,9 @@ const (
 )
 
 const (
-	DaemonKindEnvKey string = "LAZYGIT_DAEMON_KIND"
-	RebaseTODOEnvKey string = "LAZYGIT_REBASE_TODO"
+	DaemonKindEnvKey   string = "LAZYGIT_DAEMON_KIND"
+	RebaseTODOEnvKey   string = "LAZYGIT_REBASE_TODO"
+	PrependLinesEnvKey string = "LAZYGIT_PREPEND_LINES"
```

Comment:
I'm not sure I'm following this code right, can you confirm:
When we want to do an interactive rebase (with a break) onto a ref, we pass `LAZYGIT_REBASE_TODO="break"` and `LAZYGIT_PREPEND_LINES=true` and then when the lazygit daemon is invoked to update the git-rebase-todo file, that file will already have all the commits from the original branch that we're about to rebase onto the ref, and we'll then prepend the break action onto the existing TODO lines so that in lazygit we can modify the remaining TODO lines. Is that right?

If that's correct, we should have a comment explaining what we're doing because it's not immediately obvious from looking at the code. We can put that comment above the `PrependLinesEnvKey` const

Reply:
Yes, your assumption is correct. I'll introduce a comment as you suggested to give more context on the reasons behind this implementation.